src/dotnet/APIView/APIViewWeb/Managers/SamplesRevisionsManager.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.AspNetCore.Mvc.Formatters;
 using APIViewWeb.Repositories;
 using APIViewWeb.LeanModels;
+using Amazon.SecurityToken.SAML;
```

Comment:
What's this and which package is it coming from?

Reply:
I just removed that